frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -301,7 +296,12 @@ export default function SurveyEdit(): JSX.Element {
                                             <div className="flex gap-4">
                                                 <PresentationTypeCard
                                                     active={value === SurveyType.Popover}
-                                                    onClick={() => onChange(SurveyType.Popover)}
+                                                    onClick={() => {
+                                                        onChange(SurveyType.Popover)
+                                                        if (survey.schedule === 'always') {
+                                                            setSurveyValue('schedule', 'once')
```

Comment:
you can extract this and reuse below with a `SurveyType` param

Reply:
@ioannisj because the `always` schedule is only supported Widget types of surveys. so if the type is set to `always`, but they change the survey to `Popover` or `API`, we go ahead and reset the value of the `schedule`. hope it makes sense